### ISSUE TITLE
How can I choose python version with docker image?

### ISSUE THREAD

**Issue by zhizec** [apache/incubator-mxnet#12597]
How can I choose python version with docker image?  With "nvidia-docker run -ti --rm mxnet/python:gpu" I got default python 2.7. Any option to choose python version or should build specified image by myself? Thank you!

**Comment by stu1130** [comment]
Thanks for submitting the issue @zhizec 
@mxnet-label-bot[Question]
You can also post the question here [link](https://discuss.mxnet.io/) to get more help from community.

**Comment by stu1130** [comment]
@mxnet-label-bot[question]

**Comment by ThomasDelteil** [comment]
@zhizec I agree with you we should provide py2 and py3 installations, @nswamy can you please update the label to [feature request] ? Thanks!
